main.py

```diff
@@ -1,7 +1,35 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
```

Comment:
The `if __name__ == "__main__":
    main()` fragment is missing. According to the checklist, you should not add it. But be aware that without this fragment, your main() function will not be called when you run this script.